tests/mock/smartContracts.ts

```diff
@@ -1,7 +1,7 @@
 import { SAVED_ACCOUNT_1 } from "./localStorage";
 
 export const MOCK_CONTRACT_ID =
-  "C000NO6F7FRDHSOFQBT2L2UWYIZ2PU76JKVRYAQTG3KZSQLYAOKIF2WB";
+  "CAQP53Z2GMZ6WVOKJWXMCVDLZYJ7GYVMWPAMWACPLEZRF2UEZW3B636S";
```

Comment:
`StrKey.isValidContract` threw an error for the original contract ID so I updated it